client/src/components/Footer.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
Nothing wrong with having this as an extra component but just a tip for future larger projects: you'll find it easier to manage many files if you include static components within their parent component's file.